dotnet-platform-specific/RetrievalOfCpuCores/RetrievalOfCpuCores/CpuInformation.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Runtime.InteropServices;
+
+namespace RetrievalOfCpuCores;
+
+public class CpuInformation
+{
+    /// <summary>
+    /// Retrieves the number of logical processors of the CPU by using Environment.ProcessorCount.
+    /// </summary>
+    /// <returns>number of logical processors</returns>
+    public int GetProcessorsCount()
+    {
+        return Environment.ProcessorCount;
+    }
+
+    /// <summary>
+    /// Retrieves the number of logical processors of the CPU.
+    /// </summary>
+    /// <returns>number of logical processors</returns>
+    public int GetLogicalProcessors()
+    {
+        foreach (var item in new System.Management.ManagementObjectSearcher("Select * from Win32_ComputerSystem").Get())
```

Comment:
@lennart02 
These strings should be constants instead of magic strings. It helps us to avoid errors and reuse the code. There are other occurrences of this

Reply:
@lennart02 
We should declare these strings as const and reuse them. From the code I can see 2 places where we used `Select * from Win32_ComputerSystem` 
If we declare it at the top of the class e.g. `const string Win32 = Select * from Win32_ComputerSystem`, we can then reuse it. This applies to the other strings too including `NumberOfLogicalProcessors` e.t.c